config/admin_ranks.txt

```diff
@@ -11,7 +11,9 @@
 ##############################################################################################################
 # PLEASE NOTE: depending on config options, some abilities will be unavailable regardless if you have permission to use them!
 
-# KEYWORDS:
+# Follow the format below when documenting new keywords so the server tools may parse it
+
+# BEGIN_KEYWORDS
```

Comment:
this is a hack, why not just split keywords out into it's own file? this one is already pretty huge